crates/pep508-rs/src/marker/tree.rs

```diff
@@ -1067,15 +1067,11 @@ impl MarkerTree {
     /// the marker will be simplified to `sys_platform == 'linux'`.
     #[must_use]
     #[allow(clippy::needless_pass_by_value)]
-    pub fn simplify_python_versions(
-        self,
-        python_version: Range<Version>,
-        full_python_version: Range<Version>,
-    ) -> MarkerTree {
+    pub fn simplify_python_versions(self, python_version: Range<Version>) -> MarkerTree {
         MarkerTree(INTERNER.lock().restrict_versions(self.0, &|var| match var {
-            Variable::Version(MarkerValueVersion::PythonVersion) => Some(python_version.clone()),
+            // Note that `python_version` is normalized to `python_full_version`.
```

Comment:
When is it normalized? It doesn't look like it is here. Before here? later?

Reply:
I guess this is a note that this only takes `full_python_versions` because we perform the normalization upstream of here?